src/node_attributes.cpp

```diff
@@ -480,11 +480,43 @@ void KhronosObjectAttributes::serialization_info() {
   SemanticNodeAttributes::serialization_info();
   serialization::field("first_observed_ns", first_observed_ns);
   serialization::field("last_observed_ns", last_observed_ns);
-  serialization::field("mesh", mesh);
   serialization::field("trajectory_positions", trajectory_positions);
   serialization::field("trajectory_timestamps", trajectory_timestamps);
   serialization::field("dynamic_object_points", dynamic_object_points);
   serialization::field("details", details);
+
+  const auto& header = io::GlobalInfo::loadedHeader();
+  if (header.version <= io::Version(1, 0, 1)) {
+    std::vector<float> xyz;
```

Comment:
```suggestion
    io::warnOutdatedHeader(header);
    std::vector<float> xyz;
```
This will warn only once but is good to put into every deprecated function.